test/test_automl/test_estimators.py

```diff
@@ -852,3 +852,27 @@ def test_pass_categorical_and_numeric_columns_to_pipeline(
     else:
         expected_dict = {i: 'numerical' for i in range(np.shape(X)[1])}
     assert expected_dict == pipeline.named_steps['data_preprocessing'].feat_type
+
+
+@pytest.mark.parametrize("as_frame", [True, False])
+def test_autosklearn_anneal(as_frame):
+    """
+    This test makes sure that anneal dataset can be fitted and scored.
+    This dataset is quite complex, with NaN, categorical and numerical columns
+    so is a good testcase for unit-testing
+    """
+    X, y = sklearn.datasets.fetch_openml(data_id=2, return_X_y=True, as_frame=False)
+    automl = AutoSklearnClassifier(time_left_for_this_task=60, ensemble_size=0,
+                                   resampling_strategy='holdout-iterative-fit')
+
+    automl_fitted = automl.fit(X, y)
```

Comment:
Could you please pass the feature types to `fit` for numpy arrays?

Reply:
Done!